packages/e2e-test-utils/README.md

```diff
@@ -11,3 +11,528 @@ npm install @wordpress/e2e-test-utils --save-dev
 ```
 
 <br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
+
+## API
+
+### activatePlugin
+
+[src/index.js#L1-L1](src/index.js#L1-L1)
+
+Activates an installed plugin.
+
+**Parameters**
```

Comment:
Noting that this output is a bit different than we see for data stores in the existing docs, see:

https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/data/data-core-editor.md

At some point, we should consolidate for consistency.